jvmtool.go

```diff
@@ -66,24 +64,15 @@ func main() {
 	// 	logger.Info("dynamic loading status", "result", status)
 	// }
 
-	r, w := io.Pipe()
-	var wg sync.WaitGroup
-
-	wg.Add(1)
-	go func() {
```

Comment:
the goroutine here is not needed, as you are waiting below until this goroutine finishes.